packages/brisa/src/utils/file-system-router/index.ts

```diff
@@ -131,6 +131,12 @@ function getParamsAndQuery(route: string, pathname: string, url: URL) {
   );
 
   const query = { ...params, ...Object.fromEntries(url.searchParams) };
```

Comment:
Initially I wanted to delete the params inside query, but breaks 244 tests. Seems bun is expecting to have params inside query...
I also wanted to remove the Object.fromEntries(url.searchParams), given that we are looping it again, but not sure why not adding it breaks 144 tests.

I think this code could be more optimised in the future.

Reply:
It works well

<img width="1327" alt="Screenshot 2024-12-20 at 13 52 42" src="https://github.com/user-attachments/assets/a012ccd5-7100-453b-ba1b-b6cc687c7894" />